blaze-core/src/main/scala/org/http4s/blazecore/util/Http2Writer.scala

```diff
@@ -40,7 +41,7 @@ private[http4s] class Http2Writer[F[_]](
         val hs = headers
         headers = null
         tail.channelWrite(
-          HeadersFrame(None, endStream = false, hs)
+          HeadersFrame(Priority.NoPriority, endStream = false, hs)
```

Comment:
Honestly, I'm starting to think this `Priority.NoPriority` name was a huge mistake.

Reply:
What would you call it?  `Unprioritized`?

Reply:
That would be better. I was thinking `None` but maybe that's to close to `Option`?